apps/application-system/form/src/routes/Application.tsx

```diff
@@ -26,5 +26,9 @@ export const Application = () => {
   if (loading) {
     return <p>Loading</p>
   }
-  return <ApplicationForm application={data.getApplication} />
+  return (
+    <>
```

Comment:
isnt this unnecessary fragment?

Reply:
yes, left overs from me testing something